lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_bump_version_beta.rb

```diff
@@ -46,24 +64,6 @@ def self.authors
       def self.is_supported?(platform)
         platform == :android
       end
-
-      private
-
-      def self.create_config
-        @current_version = Fastlane::Helper::Android::VersionHelper.get_release_version()
-        @current_version_alpha = Fastlane::Helper::Android::VersionHelper.get_alpha_version()
-        @new_version_beta = Fastlane::Helper::Android::VersionHelper.calc_next_beta_version(@current_version, @current_version_alpha)
-        @new_version_alpha = ENV['HAS_ALPHA_VERSION'].nil? ? nil : Fastlane::Helper::Android::VersionHelper.calc_next_alpha_version(@new_version_beta, @current_version_alpha)
-      end
-
-      def self.show_config
-        vname = Fastlane::Helper::Android::VersionHelper::VERSION_NAME
-        vcode = Fastlane::Helper::Android::VersionHelper::VERSION_CODE
-        UI.message("Current version: #{@current_version[vname]}(#{@current_version[vcode]})")
-        UI.message("Current alpha version: #{@current_version_alpha[vname]}(#{@current_version_alpha[vcode]})") unless ENV['HAS_ALPHA_VERSION'].nil?
-        UI.message("New beta version: #{@new_version_beta[vname]}(#{@new_version_beta[vcode]})")
-        UI.message("New alpha version: #{@new_version_alpha[vname]}(#{@new_version_alpha[vcode]})") unless ENV['HAS_ALPHA_VERSION'].nil?
-      end
```

Comment:
👌  Nice cleanup, especially since that was using bad practices (use of instance variables in a class method context). ✨